database/feegrant.go

```diff
@@ -0,0 +1,46 @@
+package database
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	feegranttypes "github.com/cosmos/cosmos-sdk/x/feegrant"
+	"github.com/forbole/bdjuno/v2/types"
+)
+
+// SaveFeeGrantAllowance allows to store the fee grant allowances for the given block height
+func (db *Db) SaveFeeGrantAllowance(allowance *feegranttypes.MsgGrantAllowance, height int64) error {
```

Comment:
I think that instead of passing a `MsgGrantAllowance` here we should build a new type that is 

```go
type Grant struct {
   feegrant.Grant
   Heght int64
}
``` 
This way it's easier to incapsualte things here and pass the `SaveFeeGrantAllowance` a single parameter instead of multiple ones